spec/features/user_visits_homepage_spec.rb

```diff
@@ -4,6 +4,6 @@
     scenario "successfully" do
         visit root_path
 
-        expect(page).to have_css('h1', text: 'Todos')
+        expect(page).to have_css('h1', text: 'To-dos')
```

Comment:
This generally would not be part of this PR. A PR should be for one feature or some related features. It's good that you found this change but you should make a separate PR to merge this in

Reply:
Reminder to not mix changes to unrelated specs into one PR.

Reply:
Reminder again to move this to another PR

Reply:
Yeah I am going to do this now or I will forget... Just have to figure out how to remove this from this PR and move it.

Reply:
There we go, you will no longer see it )))